lib/screen/club_detail/club_detail_screen.dart

```diff
@@ -58,44 +56,62 @@ class ClubDetailScreenState extends State<ClubDetailScreen> with BackNavigatorMi
-                                ConditionalShower<List<Member>>(
-                                  data: viewModel.members,
-                                  builder: (context, members) => MemberList(
-                                    title: localization.clubDetailMemberTitle,
-                                    memberList: members,
+                          LayoutBuilder(builder: (context, constraints) {
+                            if (constraints.maxWidth > 1200) {
+                              return Row(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: [
+                                  Expanded(
+                                    child: LeftColumn(
+                                      activitySummary: viewModel.activitySummary,
+                                      admins: viewModel.admins,
+                                      members: viewModel.members,
+                                      memberStats: viewModel.memberStats,
+                                      selectedMember: viewModel.selectedMember,
+                                      setSelectedMember: viewModel.setSelectedMember,
+                                      activities: viewModel.activities,
+                                      localization: localization,
+                                    ),
                                   ),
-                                ),
-                                const SizedBox(
-                                  height: ThemeDimens.padding16,
-                                ),
-                                ConditionalShower<List<Member>>(
-                                  data: viewModel.admins,
-                                  builder: (context, admins) => MemberList(
-                                    title: localization.clubDetailAdminTitle,
-                                    memberList: admins,
+                                  const SizedBox(
+                                    width: ThemeDimens.padding32,
                                   ),
-                                ),
-                              ]),
-                              const SizedBox(
-                                width: ThemeDimens.padding32,
-                              ),
-                              Column(
-                                children: [
-                                  ConditionalShower<List<Activity>>(
-                                    data: viewModel.activities,
-                                    builder: (context, activities) => ActivityList(
-                                      title: localization.clubDetailActivityTitle,
-                                      activities: activities,
+                                  Expanded(
+                                    child: RightColumn(
+                                      selectedActivity: viewModel.selectedActivity,
```

Comment:
This imlicates an `Activity` but it's an `int` change to `selectedIndex` or something